src/main/scala/com/vngrs/etl/Combine.scala

```diff
@@ -0,0 +1,49 @@
+package com.vngrs.etl
+
+import org.apache.spark.rdd.RDD
+
+import scala.reflect.ClassTag
+
+/**
+  * [[com.vngrs.etl.Combine]] is an interface for combining two [[com.vngrs.etl.Pipeline]]s into one.
+  *
+  * @tparam A Type of Input [[com.vngrs.etl.Pipeline]]
+  * @tparam B Type of Output [[com.vngrs.etl.Pipeline]]
+  */
+trait Combine[A, B] {
```

Comment:
I still think this one is a bad abstraction. Can you show me any case it is useful?